### PR TITLE
Fix deeplab tests - keras 2

### PR DESCRIPTION
**Pull request by divyashreepathihalli**

### CODE REVIEW

**Comment by divyashreepathihalli** [comment]
https://github.com/keras-team/keras/pull/18933

File: keras_cv/backend/random.py [hunk 1]
```diff
@@ -19,43 +19,28 @@
...
 
-    def next(self, ordered=True):
-        if keras_3():
-            return self._seed_generator.next(ordered=ordered)
-        else:
+        def next(self, ordered=True):
             self._current_seed[0] += 1
             return self._current_seed[:]
 
+        def get_config(self):
```
Comment (sampathweb) [inline keras_cv/backend/random.py:30]: You might need a `from_config` as well.

Comment (divyashreepathihalli) [reply keras_cv/backend/random.py:30]: yes, thats right, I will add that

**Review by y-vectorfield — COMMENTED** [review]
This modification needs to solve [this problem](https://github.com/keras-team/keras-cv/issues/2238).

**Comment by sampathweb** [comment]
Addressed in #2243 to combine test failures fixes so we can test them together.  Closing this PR.

**Pull request closed without merging**